### ISSUE TITLE
Unable to launch an app

### ISSUE THREAD

**Issue by rainder** [nathanjohnson320/io_arm#1]
```
pi@raspberrypi ~ $ iojs -v
iojs: /usr/lib/arm-linux-gnueabihf/libstdc++.so.6: version `GLIBCXX_3.4.20' not found (required by iojs)
iojs: /lib/arm-linux-gnueabihf/libc.so.6: version `GLIBC_2.16' not found (required by iojs)
```

Fresh install of raspbian/wheezy
Ay ideas?

**Comment by nathanjohnson320** [comment]
I had to update my RPI to use the Jessie sources in order to compile. Try upgrading and installing glibc

**Comment by nathanjohnson320** [comment]
http://www.raspberrypi.org/forums/viewtopic.php?f=66&t=47944

**Comment by nathanjohnson320** [comment]
Since libc is missing I'd try this as well
sudo apt-get install libc6

**Comment by fivdi** [comment]
The armv6l binaries at iojs.org run on the stock version of Raspbian Wheezy. For example the v1.2.0 binaries [here](https://iojs.org/download/release/v1.2.0/) run on January 2015 version of Raspbian Wheezy. It's not necessary to update to Jessie or install any additional software. Maybe io_arm could use these binaries?

**Comment by nathanjohnson320** [comment]
I think with them building a binary like that there's no need for io_arm. All you have to do with the binary is download it and add it to the path which is just as easy as the io_arm package.

**Comment by fivdi** [comment]
Maybe you're right. The io_arm way seems a little easier though.

**Comment by nathanjohnson320** [comment]
Yeah I like dealing with packages more than binaries since you don't have to deal with the path.

Do you happen to know how they're compiling their binary? If I can duplicate that then I can turn it into a debian package after I've run a "make" without upgrading to jessie.

**Comment by fivdi** [comment]
Sorry, I'm afraid I don't know how they're building the binaries. Can you not just take theirs?

**Comment by nathanjohnson320** [comment]
I might be able to. I've never built a debian package by hand though I've always used checkinstall. I'll give it a shot this weekend.
